recipes/_install-windows.rb

```diff
@@ -28,10 +28,15 @@
 dd_agent_msi = dd_agent_version ? "ddagent-cli-#{dd_agent_version}.msi" : 'ddagent-cli.msi'
 temp_file = ::File.join(Chef::Config[:file_cache_path], 'ddagent-cli.msi')
 
+package_retries = node['datadog']['agent_package_retries']
+package_retry_delay = node['agent_package_retry_delay']
```

Comment:
Actually missed this: should be `node['datadog']['agent_package_retry_delay']`

Reminds me that I'd appreciate a few simple spec tests on this :)